src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeBase.java

```diff
@@ -0,0 +1,152 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.wpi.first.math.util.Units;
+
+import java.util.LinkedList;
+
+import static edu.greenblitz.tobyDetermined.RobotMap.TelescopicArm.Elbow.STARTING_ANGLE_RELATIVE_TO_GROUND;
+
+public class NodeBase  {
+    private NodeArm REST_ABOVE_BELLY;
+    private NodeArm INTAKE_GRAB_CUBE_POSITION;
+    private NodeArm CONE_HIGH;
+    private NodeArm CONE_MID;
+    private NodeArm CUBE_HIGH;
+    private NodeArm CUBE_MID;
+    private NodeArm LOW;
+    private NodeArm ZIG_HAIL;
+    private NodeArm INTAKE_GRAB_CONE_POSITION;
+    private NodeArm PRE_CONE_DROP;
+    private NodeArm POST_CONE_DROP;
+    private NodeArm currentNode;
+    private int i = 1;
```

Comment:
Using `i` as a variable name outside of a simple for-loop is considered to be a bad practice (actually true for all single-letter variable/param names)

Reply:
changed name